src/app/components/GoogleSheetsInput.svelte

```diff
@@ -73,10 +73,11 @@
    *
    */
   function getSpreadsheetId() {
-    googleSheetsURL = document.getElementById("googleSheetsURL").value;
+    // TODO: should not use id; should make use of bindings
+    googleSheetsURL = document.getElementById("input-googleSheetsURL").value;
 
     const googleSheetRegExp = RegExp(
-      "http(s?)://docs.google.com/spreadsheets/[a-z]+/[a-zA-z0-9]+/[a-zA-z0-9#=]+"
+      "http(s?)://docs.google.com/spreadsheets/[a-z]{1}/[\\w-]+/[\\w#=-]+"
```

Comment:
Out of curiosity, what's does this different pattern do compared to the old pattern?

Reply:
This part `[a-zA-z0-9]+` from the old one was a little more strict in that it didn't allow underscores or dashes. Other changes are just to make it a bit simpler!